tests/test_fixtures.py

```diff
@@ -138,32 +141,32 @@ def test_route_handler_fake_bad_path() -> None:
 def test_fake_element_not_installed() -> None:
     """We don't request the fixture so it isn't available."""
     with pytest.raises(NameError):
-        Element  # noqa
+        Element # noqa
 
 
-def test_fake_element_installed(fake_element) -> None:
+def test_fake_element_installed(fake_element: function) -> None:
     """Element is available as ``fake_element`` installed it."""
-    Element  # noqa
+    Element # noqa
 
 
-def test_fake_element_find_element(fake_document, fake_element) -> None:
+def test_fake_element_find_element(fake_document: FakeDocument, fake_element: function) -> None:
```

Comment:
What's your feeling on my commitment to typing? Too much?

My plan is: actual examples don't have to use typing. Perhaps even - never use typing. But the PSC itself does aim for good typing coverage.

Reply:
These are the type of issues I still need to see if I can make mypy happy. 
In general I am in favor of typing, but I think in a lot of cases we will have mypy break for examples when we do `Element` it complains that its not defined.

Your idea is great, keeping the examples easy to follow for new folks is great and enforcing typing there will make things more complicated 😄